docs/azure-quantum/access.md

```diff
@@ -0,0 +1,23 @@
+# Setting up access and authentication to Azure Quantum
+
+[Azure Quantum](https://docs.microsoft.com/azure/quantum/overview-azure-quantum) is Microsoft's cloud service for running quantum computing programs or solving optimization problems and is currently in [Public Preview](https://cloudblogs.microsoft.com/quantum/2021/02/01/azure-quantum-preview/). You can try Azure Quantum for free today and submit quantum programs to our quantum partners and technologies. Follow the simple steps below to set up access and start sending quantum programs with Cirq to IonQ or Honeywell.
```

Comment:
Change "to our partners" since "our" is ambiguous whether it means cirq's partner's or Microsoft partners.

I would say "submit quantum programs to Azure quantum partners and technologies."